packages/flutter/test/material/buttons_test.dart

```diff
@@ -1191,4 +1192,45 @@ void main() {
     final Material material = tester.widget<Material>(rawButtonMaterial);
     expect(material.color, const Color(0xff00ff00));
   });
+
+  group('feedback', () {
+    FeedbackTester feedback;
+
+    setUp(() {
+      feedback = FeedbackTester();
+    });
+
+    tearDown(() {
+      feedback?.dispose();
+    });
+
+    testWidgets('MaterialButton with disabled feedback', (WidgetTester tester) async {
+      await tester.pumpWidget(Directionality(
+        textDirection: TextDirection.ltr,
+        child: MaterialButton(
+          onPressed: () {},
+          enableFeedback: false,
+        ),
+      ));
+      await tester.tap(find.byType(MaterialButton), pointer: 1);
+      await tester.pump(const Duration(seconds: 1));
+      expect(feedback.clickSoundCount, 0);
+      expect(feedback.hapticCount, 0);
+    });
+
+    testWidgets('MaterialButton with enabled feedback', (WidgetTester tester) async {
```

Comment:
This feels like it tests the default behavior (that `enableFeedback` is set to `true` by default). You should probably add a test where `enableFeedback` is explicitly set to `true`. ie.
```
      await tester.pumpWidget(Directionality(
        textDirection: TextDirection.ltr,
        child: MaterialButton(
          onPressed: () {},
          enableFeedback: true, // add a test for this case
        ),
      ));
```

Reply:
Done